src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -74,12 +155,12 @@ The most common causes of a poor CLS are:
   </figcaption>
 </figure>
 
-### History
+#### History
 
 In the early days of the web, developers would add `width` and `height` attributes to their `<img>` tags to ensure sufficient space was allocated on the page before the browser started fetching images. This would minimize reflow and re-layout.
 
 ```html
-<img src="puppy.jpg" width="640" height="360" alt="Puppy with balloons" />
+<img src="puppy.jpg" width="640" height="360" alt="Puppy with balloons">
 ```
 
 You may notice `width` and `height` above do not include units. These "pixel" dimensions would ensure a 640x360 area would be reserved. The image would stretch to fit this space, regardless of whether the true dimensions matched or not.
```

Comment:
Might be worth clarifying this and refer to the concept of aspect ratio, as that's what the unitless values provide.

Reply:
It's coming in a few paragraphs. I left this part of the doc largely untouched and am comfortable with that.